pkg/log/context.go

```diff
@@ -9,6 +9,17 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+//log formats as defined by LOG_FORMAT env variable
+const (
+	LogFormatText = "text"
+	LogFormatJson = "json"
+)
+
+type Config struct {
+	LogLevel  string `envconfig:"LOG_LEVEL" default:"info"`
+	LogFormat string `envconfig:"LOG_FORMAT" default:"text"`
```

Comment:
default - json, no?

Reply:
After talking with Yoni: I will change the default in the code to json and leave the configuration variable only in deploy/assisted-service where it will be set to text as default